### PR TITLE
Publish docs from docs-devel

### PR DESCRIPTION
**Pull request by orndorffgrant**
## Why is this needed?

This publishes docs that were staged in `docs-devel` until release. Now that v34 is out we can publish them.

## Test Steps

These were already reviewed once before merging into `docs-devel`. Check the PR build to make sure the new docs still look correct.

---

- [ ] *(un)check this to re-run the checklist action*

### CODE REVIEW

File: dev-docs/howtoguides/run_dep8_tests.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# How to run dep8 tests
+
+Our package currently supports [dep8
```
Comment (s-makin) [inline dev-docs/howtoguides/run_dep8_tests.md:3]: Which package?

File: dev-docs/howtoguides/run_dep8_tests.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+tests]( https://salsa.debian.org/ci-team/autopkgtest/-/blob/master/doc/README.package-tests.rst).
+Those tests can be found under `debian/tests/usage` and although the test is running
+some Pro client commands, the idea is to verify if any modification to our package dependencies
```
Comment (s-makin) [inline dev-docs/howtoguides/run_dep8_tests.md:6]: This is a bit unclear - it's simultaneously referring to multiple tests and one test, and "some commands" but also "this command". Could we make this more specific?

File: dev-docs/howtoguides/run_dep8_tests.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+sudo apt install autopkgtest
+```
+
+2. Run the following command:
+
+```shell
+autopkgtest -U --shell-fail . -- lxd ubuntu:xenial
+```
+
+Note that you can run this command on any release we support, not only Xenial.
```
Comment (s-makin) [inline dev-docs/howtoguides/run_dep8_tests.md:26]: Should it be run against all releases, or is it usual just to test against one?

File: docs/howtoguides/how_to_know_if_system_affected_by_a_list_of_cves.rst [hunk 4]
```diff
@@ -0,0 +1,55 @@
+.. _pro-check-list-cves:
+
+How to know if a system is affected by this list of CVEs?
```
Comment (s-makin) [inline docs/howtoguides/how_to_know_if_system_affected_by_a_list_of_cves.rst:3]:
```suggestion
Check if a system is affected by a list of CVEs
```

File: docs/howtoguides/how_to_know_if_system_affected_by_a_list_of_cves.rst [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+If you have a list of `Common Vulnerabilities and Exposure <cve_>`_ (CVEs) and `Ubuntu Security Notice <usn_>`_ (USNs) and want to check if your Ubuntu system is affected by it, you can check using the ``u.pro.security.fix.cve.plan.v1`` API endpoint as we'll show you in this guide.
+
+.. note::
+    The ``u.pro.security.fix.cve.plan`` API is provided as a part of the Ubuntu Pro Client (``pro``), which is a security tool for Ubuntu systems. The Ubuntu Pro Client comes pre-installed on every Ubuntu system. You can run ``pro help`` in your terminal window to see a list of the ``pro`` services and commands available.
+
+Using the ``pro.fix.cve.plan`` API
+----------------------------------
+
+To check if your system is affected by a list of CVEs, you need to use the ``u.pro.security.fix.cve.plan.v1`` API endpoint since the
+``pro fix --dry-run`` CLI command is only used to check individual CVEs. This endpoint will output a JSON blob containing the current status of each CVE, as can be seen :ref:`in the endpoint documentation <cve-execute-api-v1>`.
```
Comment (s-makin) [inline docs/howtoguides/how_to_know_if_system_affected_by_a_list_of_cves.rst:17]: We're getting a build error on RTD related to this link - does the anchor `cve-execute-api-v1` exist in the docs?

File: docs/howtoguides/how_to_know_if_system_affected_by_a_list_of_cves.rst [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+To better visualise the current status of each CVE from the JSON output we can use a ``jq`` filter.
+The ``jq`` command can parse JSON data directly in the terminal, creating a more readable output. To learn more about it, refer to `the jq manpage <https://manpages.ubuntu.com/manpages/man1/jq.1.html>`_
```
Comment (s-makin) [inline docs/howtoguides/how_to_know_if_system_affected_by_a_list_of_cves.rst:20]: This link is being flagged by the linkchecker as "broken" - we might need to add it as an exception in the ignore list (in conf.py)

File: docs/howtoguides/how_to_know_if_system_affected_by_a_list_of_cves.rst [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+and **CVE-2017-9233**.
+
+We make use of the plan API by running the following command:
+
+.. code-block:: bash
+
+    pro api u.pro.security.fix.cve.plan.v1 --data '{"cves": ["CVE-2020-28196", "CVE-2020-15180", "CVE-2017-9233"]}' \ 
+        | jq -r '.data.attributes.cves_data.cves[] | "\(.title) (\(.description)) - Current Status: \(.current_status)"'
+
+This command will return an output that will have the following structure:
```
Comment (s-makin) [inline docs/howtoguides/how_to_know_if_system_affected_by_a_list_of_cves.rst:40]:
```suggestion
This command returns output with the following structure:
```

**Review by s-makin — COMMENTED** [review]
Can't check the PR preview in RTD because of a build failure, but here's my initial suggestions

**Comment by renanrodrigo** [comment]
@orndorffgrant this now contains the commit from @dheyay that is related only to v35 :D

please please rediscussion of process in Hague

**Comment by orndorffgrant** [comment]
This is replaced by
* #3355
* #3356
* #3357

All comments are addressed in the new PRs.
Closing this one and deleting docs-devel forever.

**Pull request closed without merging**